index.bs

```diff
@@ -73,51 +67,31 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The Accelerometer's associated <a>Sensor subclass</a>
-is the {{Accelerometer}} class.
-
-The Accelerometer's associated <a>SensorReading subclass</a>
-is the {{AccelerometerReading}} class.
-
-The Accelerometer has a <a>default sensor</a>,
-which is the device's main accelerometer sensor.
+The accelerometer's associated {{Sensor}} is the {{Accelerometer}} class.
```

Comment:
Other specs use this boilerplate definition:

The accelerometer's associated {{Sensor}} subclass is the {{Accelerometer}} class.

Should perhaps align with that.